cuda_phase-retrieval/run.py

```diff
@@ -9,45 +9,49 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument("--image", help="Image path")
-parser.add_argument("--step", help="number of iterations (default 20)", type=int, default=20)
-parser.add_argument("--beta", help="beta (default 0.8)", type=float, default=0.8)
-parser.add_argument("--type", help="Hybrid, Input-Ouput, or Output-Output", default="hybrid")
+parser.add_argument("--mask", help="Mask value (default 1). This will automatically create array of the specified value with the same size of the input image", type=float, default=1)
+parser.add_argument("--step", help="Number of iterations (default 20)", type=int, default=20)
+parser.add_argument("--beta", help="Beta (default 0.8)", type=float, default=0.8)
+parser.add_argument("--mode", help="Hybrid, Input-Ouput, or Output-Output", default="hybrid")
```

Comment:
`argparse` provides the option `choices` to restrict the input options: https://github.com/ComputationalRadiationPhysics/xeus-cling-cuda-container/blob/7138519646ee9d2918bf3bb006830cfdec981889/dev_container.py#L31
It generates automatically the help string and controls the argument parameter. You can use it to replace your manual check.